webapp/src/modules/item/reducer.spec.ts

```diff
@@ -207,19 +230,42 @@ describe.each([
 
 describe('when reducing the successful action of fetching an item', () => {
   const requestAction = fetchItemRequest(item.contractAddress, item.itemId)
-  const successAction = fetchItemSuccess(item)
+  let successAction = fetchItemSuccess(item)
 
-  const initialState = {
+  let initialState = {
     ...INITIAL_STATE,
     data: { anotherId: anotherItem },
     loading: loadingReducer([], requestAction)
   }
 
-  it('should return a state with the the loaded items with the fetched item and the loading state cleared', () => {
-    expect(itemReducer(initialState, successAction)).toEqual({
-      ...INITIAL_STATE,
-      loading: [],
-      data: { ...initialState.data, [item.id]: item }
+  describe('and the fetched item is not in the state', () => {
+    it('should return a state with the the loaded items with the fetched item and the loading state cleared', () => {
```

Comment:
What do you think about this change? Is it easier to read?
```suggestion
    it('should return a state with the loaded items, the fetched item and the loading state cleared', () => {
```